src/s3.js

```diff
@@ -35,11 +35,39 @@ const listObjects = async (prefix) => {
     StartAfter: prefix,
   };
   const result = await s3.listObjectsV2(params).promise();
-  const { Contents: objects } = result;
+
```

Comment:
I made our existing `listObjects` function more robust. The S3 API `listObjectsV2` can only retrieve up to 1000 objects:
> Returns some or all (up to 1,000) of the objects in a bucket with each request.

Reference: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#listObjectsV2-property

We won't run into the problem for a while, but we will when we have >1000 files in a given folder. So this code will keep looping and calling `listObjectsV2` until all the files are retrieved. If `NextContinuationToken` exists, then there are more files to retrieve; otherwise, we're done.